config/templates/csv-template.yaml

```diff
@@ -28,16 +28,16 @@ spec:
   - type: SingleNamespace
     supported: true
   - type: MultiNamespace
-    supported: false
+    supported: true
   - type: AllNamespaces
     supported: false
   install:
     strategy: deployment
     spec:
       clusterPermissions: []
         # Rules will be added here by the generate-operator-bundle.py script.
-      deployments: []
-        # Deployment spec will be added here by the generate-operator-bundle.py script.
+      deployments:
+      - name: pagerduty-operator
```

Comment:
This delta shouldn't be needed, but is harmless. The whole `deployments` list gets [overwritten](https://github.com/openshift/pagerduty-operator/pull/144/files?file-filters%5B%5D=.py&file-filters%5B%5D=.yaml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile#diff-e12cb709d3181a1ca667b183451c41bbb410dd055344a17d1c0bc981430fde16R289) anyway.